### DISCUSSION TITLE
[feat] display trace times as histograms/chart

### DISCUSSION THREAD

**Discussion by baggiponte** [darrenburns/posting#139] [Ideas]
Ciao! Been looking at posting to fully replace postman. One thing that I find helpful is to see the trace displayed not just as a time unit, like so:

<img width="394" alt="image" src="https://github.com/user-attachments/assets/4b8bd2d5-7185-48e3-ab12-29ac3fd5821b">

But as a chart:

<img width="452" alt="image" src="https://github.com/user-attachments/assets/69e3071f-80f9-460f-bc91-0825a6c2109c">

Would you deem it a worthy addition? 😊

**Comment by darrenburns** [comment]
Yep, that'd be cool. The "trace" tab hasn't had much love :)
